src/Components/Books.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Book = () => (
```

Comment:
- [ ] According to the requirements for this project, a single Book component should the book data as a prop and renders the title and the author.

Example: 
```js
const Book = ({ title, author}) => (

<div className="Book">
     <div>
           <h3>Action</h3>
           <h2>{title}</h2>
           <p>{author}</p>
     </div>
    // other code snippets
 </div>
)
```

Reply:
Thank you for the advice. I changed it and pass it as props.